src/components/Modals/NewCourse.vue

```diff
@@ -1,24 +1,102 @@
 <template>
   <div class="newCourse">
     <div v-if="!isOnboard" class="newCourse-text">{{ text }}</div>
+    <!-- TODO: for some reason this breaks the dropdown <div v-if="selected" class="newCourse-name newCourse-requirements-container">{{ selectedCourse }}</div> -->
```

Comment:
What is going on here?

Reply:
So this is the line of code regarding this visual inconsistency. For some reason when I add it, the next time you go into the NewCourse modal the dropdown for classes doesn't show up. So, for now I just sacrificed the visuals for a working dropdown since I'm not 100% sure what type of logic is going on in the `autocomplete` function anyway.
![image](https://user-images.githubusercontent.com/55263191/98966386-358f0280-24d9-11eb-932c-43be3ba1654c.png)

Reply:
Oh sure that's fine! The autocomplete is very bad and needs to be refactored, so not your fault at all.

Speaking of though, could you make a Monday item titled something like "New Add Course Modal Figma inconsistencies" or something like that, and put in the description any unresolved problems from this PR? So this and anything else that might come up later.